code/GameManager.cs

```diff
@@ -131,6 +134,7 @@ public override void OnVoicePlayed( IClient client )
 
 	public override void PostLevelLoaded()
 	{
+		MapTimer = 60 * TimeLimitMinutes;
```

Comment:
If the player changes the convar, maptimer is never going to get updated. Can we just use `TimeLimitMinutes`? Does TimeUntil work as a replicated convar?

Reply:
I don't think `[Convar]` works with RealTimeUntil/TimeUntil or on structs in general that, otherwise that's what I would have done.  That and you wouldn't be able to do it in minutes and when you wanted to change it in the console you'd have to type it in seconds and not minutes. I tried anyway and got a cool neat error that I could replicate in a fresh gamemode.

![image](https://user-images.githubusercontent.com/66768086/211603234-58cc2ba1-34a0-44b8-b360-bb9a8d63fe3d.png)
![image](https://user-images.githubusercontent.com/66768086/211603265-88babe02-5ed5-4e43-8f95-d2f62c5f2779.png)

Reply:
We can make a CondCmd for admins to override the time

Reply:
You could just do an OnChange callback into a TimeUntil networked variable. Check out the voice chat convar, same thing except on change you update the networked variable.